mobile/lib/main.dart

```diff
@@ -63,12 +63,22 @@ import 'features/stable/stable_value_change_notifier.dart';
 final GlobalKey<NavigatorState> _rootNavigatorKey = GlobalKey<NavigatorState>(debugLabel: 'root');
 final GlobalKey<NavigatorState> _shellNavigatorKey = GlobalKey<NavigatorState>(debugLabel: 'shell');
 
-void main() {
+void main() async {
```

Comment:
Does this work? My limited flutter knowledge says you'd need

```suggestion
Future<void> main() async {
```

Reply:
Also, can the main be async?

Reply:
yes, it works and all the examples in the official Dart documentation show it like this:

https://dart.dev/codelabs/async-await#working-with-futures-async-and-await